modules/third-party/tomcat-jasper/patches/LPS-91105.patch

```diff
@@ -0,0 +1,269 @@
+diff --git a/org/apache/jasper/JspCompilationContext.java b/org/apache/jasper/JspCompilationContext.java
```

Comment:
@dantewang @yuhai 
Are these patches required by old patched of glassfish jasper or these are new requirements?

Tina

Reply:
@tinatian Some changes are migrated from Glassfish Jasper patch, some are new, but they are mainly about class loading issues.